src/client/styles/_topbar.scss

```diff
@@ -67,6 +67,24 @@
     margin-right: 10px;
 }
 
+#run-btn .main-text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+#run-btn .main-text .inactive {
+    position: absolute;
+    display: hidden;
```

Comment:
"hidden" isn't a valid value for "display." You're probably thinking of "none," but "display: none" doesn't play nicely with animations like this, since it can't be animated, so the element disappears immediately as soon as the animation begins.

Reply:
You're right, I got mixed up! Thanks for pointing that out.